### PR TITLE
Team page update

### PR DESCRIPTION
**Pull request by christymelucci**

### CODE REVIEW

**Comment by christymelucci** [comment]
Alex, after you approve the addition of Robin Mejia, I will check to see if there are other Team Page updates to be made before I complete the merge this week. Please indicate if this is time sensitive and I will merge right away.

**Review by capnrefsmmat — APPROVED** [review]
Looks good, thanks. Not time-sensitive, so this is okay to go in the next site update

**Pull request merged**
